src/content/docs/apm/agents/php-agent/troubleshooting/troubleshooting-resources.mdx

```diff
@@ -0,0 +1,137 @@
+---
+title: PHP Agent Troubleshooting Resources
+type: troubleshooting
+tags:
+  - Agents
+  - PHP agent
+  - Troubleshooting
+redirects:
+  - /docs/agents/php-agent/troubleshooting/troubleshooting-resources
```

Comment:
```suggestion
  - /docs/agents/php-agent/troubleshooting/troubleshooting-resources
  - /docs/agents/php-agent/troubleshooting
```
This way it acts as a landing page for anyone who just searches for `https://docs.newrelic.com/docs/apm/agents/php-agent/troubleshooting/` without having to specify the exact page name.